components/Profile/Orders/Card.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <BaseCard :title="title">
+    <div v-if="orders.length">
+      <div
+        v-for="(order, index) in orders"
+        :key="order.id"
+        class="min-h-[100px] border-gray-300 pt-2.5"
+        :class="{
+          'border-b pb-2.5': index !== orders.length - 1,
+          '!pt-0': index === 0,
+          '!pb-0': index === orders.length - 1,
+        }"
```

Comment:
Fancy 😄  Vue styling with Tailiwind seems much more JS driven than without it. You used indexes here to apply classes, while normally we would use selectors like `:first-child`, `:last-child` etc. right?

Also, could it be something like this?
```js
'!pt-0': index === 0 || index === orders.length -1
```

Maybe it would be a nice touch to make those classes logic a `computed()`? 🤔

Reply:
I will see what can I do, if the First/Last child will work I will be happy 🤣 I forgot to try it tbh 🤭🤭

Reply:
Done